### PR TITLE
Casmcms 7622

### PR DESCRIPTION
**Pull request by jsl-hpe**
## Summary and Scope

Update charts to use new image schema versioning for cray basechart 7.x.x.

## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves CASMCMS-7622

## Testing

Ongoing; rocket.

### Tested on:

  * rocket
  * local dev system builds

### Test description:

- helm deploy upgrade performed.

## Risks and Mitigations

- NA

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable
- [x] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: kubernetes/csmsshkeys/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,30 @@
-apiVersion: v1
+annotations:
+  artifacthub.io/images: |
+    - name: cfs-trust
```
Comment (rbak-hpe) [inline kubernetes/csmsshkeys/Chart.yaml:3]: I'm guessing this name should be csm-ssh-keys

**Review by rbak-hpe — APPROVED** [review]

**Review by rkleinman-hpe — CHANGES_REQUESTED** [review]
GH action linting errors

File: kubernetes/csmsshkeys/Chart.yaml (hunk 1 above)
Comment (jsl-hpe) [reply kubernetes/csmsshkeys/Chart.yaml:3]: You have guessed incorrectly.

Comment (jsl-hpe) [reply kubernetes/csmsshkeys/Chart.yaml]: Correct! Updated.

**Review by rkleinman-hpe — APPROVED** [review]

**Pull request merged**